### PR TITLE
add error prone to the build.

### PR DESCRIPTION
**Pull request by rkirov**
Fixes existing error prone issues.

Could not figure out how to make error prone warnings errors. So for now
just print the warnings during building.

### CODE REVIEW

**Comment by rkirov** [comment]
It works for me locally but fails on travis - https://travis-ci.org/angular/clutz/builds/208821370
No idea why :(

**Comment by dpurp** [comment]
Fixes #458

File: build.gradle [hunk 1]
```diff
@@ -72,4 +75,4 @@ test {
```
Comment (dpurp) [inline build.gradle]: do you need a classpath/runtime dependency on error-prone as well?

https://github.com/google/error-prone/blob/master/examples/gradle/build.gradle#L9

**Comment by rkirov** [comment]
according to https://plugins.gradle.org/plugin/net.ltgt.errorprone this should be enough for gradle 2.1+

**Comment by dpurp** [comment]
I wonder if this is the issue:
http://stackoverflow.com/questions/37140114/gradle-check-fails-in-travis-ci

**Comment by rkirov** [comment]
Gradle wrapper fixed the issue and it appear that Travis is caching the extra downloads (gradle 3.2), so ready to PTAL.

**Pull request merged**